### PR TITLE
class_center_sample

### PR DESCRIPTION
**Pull request by teckno**

### CODE REVIEW

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

File: ivy/functional/frontends/paddle/nn/functional/common.py [hunk 1]
```diff
@@ -109,3 +109,8 @@ def interpolate(
 def linear(x, weight, bias=None, name=None):
     weight = ivy.swapaxes(weight, -1, -2)
     return ivy.linear(x, weight, bias=bias)
+
+@to_ivy_arrays_and_back
+@with_supported_dtypes({"2.5.1 and below": ("float32", "float64")}, "paddle")
+def class_center_sample ( label, num_classes, num_samples, group=None ):
+    return ivy.class_center_sample( label, num_classes, num_samples, group=group )
```
Comment (hirwa-nshuti) [inline ivy/functional/frontends/paddle/nn/functional/common.py:116]: Ivy doesn't have this function implemented on our API

File: ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py [hunk 2]
```diff
@@ -416,3 +416,33 @@ def test_linear(
...
+    *,
+    dtype_x_weight_bias,
+    on_device,
+    fn_tree,
+    backend_fw,
+    frontend,
+    test_flags,
+):
+    dtype, x, weight, bias = dtype_x_weight_bias
+    weight = ivy.swapaxes(weight, -1, -2)
```
Comment (hirwa-nshuti) [inline ivy_tests/test_ivy/test_frontends/test_paddle/test_nn/test_functional/test_common.py:437]: Ivy dependency should be removed

**Review by hirwa-nshuti — CHANGES_REQUESTED** [review]
Hey @teckno 

Check the comments below

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Pull request closed without merging**